lib/publiccloud/provider.pm

```diff
@@ -39,9 +39,9 @@ has ssh_key => get_ssh_private_key_path();
 =cut
 
 sub init {
-    my ($self) = @_;
+    my ($self, %args) = @_;
     $self->create_ssh_key();
-    $self->place_ssh_config();
+    $self->place_ssh_config() unless ($args{skip_ssh_local_config});
```

Comment:
I'd just use variable instead of adding complexity to the code.

Reply:
Variable like @asmorodskyi is suggesting above? Or an openQA setting? If it's the second one, we already have settings which we can use to decide, but I'm not sure you guys would like a combination of `get_var()` calls in this `init()` method. 

First approach I used yesterday was to have the module removing `.ssh/config` after it was created. That doesn't seem too elegant IMHO, so I think it's best not to create it unless it's going to be used.